contrib/labstack/echo/echotrace.go

```diff
@@ -27,30 +27,26 @@ func Middleware(opts ...Option) echo.MiddlewareFunc {
 			fn(cfg)
 		}
 		log.Debug("contrib/labstack/echo: Configuring Middleware: %#v", cfg)
+
 		return func(c echo.Context) error {
 			request := c.Request()
 			resource := request.Method + " " + c.Path()
 			opts := []ddtrace.StartSpanOption{
-				tracer.ServiceName(cfg.serviceName),
-				tracer.ResourceName(resource),
-				tracer.SpanType(ext.SpanTypeWeb),
-				tracer.Tag(ext.HTTPMethod, request.Method),
-				tracer.Tag(ext.HTTPURL, request.URL.Path),
 				tracer.Measured(),
 			}
-
 			if !math.IsNaN(cfg.analyticsRate) {
 				opts = append(opts, tracer.Tag(ext.EventSampleRate, cfg.analyticsRate))
 			}
-			if spanctx, err := tracer.Extract(tracer.HTTPHeadersCarrier(request.Header)); err == nil {
-				opts = append(opts, tracer.ChildOf(spanctx))
-			}
+
 			var finishOpts []tracer.FinishOption
 			if cfg.noDebugStack {
-				finishOpts = append(finishOpts, tracer.NoDebugStack())
+				finishOpts = []tracer.FinishOption{tracer.NoDebugStack()}
 			}
-			span, ctx := tracer.StartSpanFromContext(request.Context(), "http.request", opts...)
-			defer func() { span.Finish(finishOpts...) }()
+
```

Comment:
Same here, and everywhere else. https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines